components/Common/PokemonList.tsx

```diff
@@ -0,0 +1,25 @@
+import { useAuth } from "@/context/AuthContext";
+import { Pokemon } from "@/types/types";
+import React from "react";
+import PokemonCard from "../Dashboard/PokemonCard";
+
+const PokemonList = ({ list }: { list: Pokemon[] }) => {
```

Comment:
Extract out props in a separate interface file and prefer using `function` for components